### PR TITLE
Fix missing 'Landing pad' reports for some stations.

### LINKED ISSUE #622: Landing pad report not firing
## EDDI version in which issue found
3.0.1-b4

## Steps to reproduce

1. Obtain docking permission at any of the supported types of port (Coriolis, Orbis, Ocellus, Asteroid Base).

## Expected
The "Landing pad report" script runs and provides directions.

## Observed
Silence

## Investigation
I added the following to the "Docking granted" Cottle script:
```
...
{set station to StationDetails(event.station)}
{if station is null: Station is null. |else: {dump station}}
...
```

and consistently got `[]` for the dump.

### PR DESCRIPTION
**Pull request by Tkael**
Fixes #622 & implements new `stationtype` variable per #347.

### CODE REVIEW

**Comment by Tkael** [comment]
Ack. I'll look at that build error this evening.

**Comment by Tkael** [comment]
Placing on-hold to add appropriate resx for station models

**Comment by Tkael** [comment]
Resx added.

File: DataDefinitions/StationModels.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Utilities;
+
+namespace EddiDataDefinitions
+{
+    /// <summary>
+    /// Crime types
```
Comment (richardbuckle) [inline DataDefinitions/StationModels.cs]: Aaaaand this is why I don't like docstrings. They get stale and tell lies. I might just nuke the lot of them, they are completely redundant anyway.

Comment (Tkael) [reply DataDefinitions/StationModels.cs]: woops. well, I guess you know where I found my template. :-)

**Pull request merged**